docs/api/sorting_and_filtration.rst

```diff
@@ -0,0 +1,610 @@
+                                        }
+                                    }
+                                },
+                                {
+                                    "code": "sticker_size_7",
+                                    "translations": {
+                                        "en_US": {
+                                            "locale": "en_US",
+                                            "id": 6,
+                                            "value": "7\""
+                                        }
+                                    }
+                                }
+                            ],
+                            "_links": {
+                                "self": {
+                                    "href": "\/api\/v1\/products\/sticker_size"
+                                }
+                            }
+                        }
+                    ],
+                    "averageRating": 0,
+                    "images": [
+                        {
+                            "id": 43,
+                            "type": "main"
+                        },
+                        {
+                            "id": 44,
+                            "type": "thumbnail"
+                        }
+                    ],
+                    "_links": {
+                        "self": {
+                            "href": "\/api\/v1\/products\/e77f129f-5921-3ad2-88bd-f27b59aad037"
+                        }
+                    }
+                }
+            ]
+        }
+    }
+
+How to filter resources?
+------------------------
+
+Let's assume that you want to find all products which contain the word ``sticker`` in the name. In this case you should call
+the ``/api/v1/products/`` endpoint with the ``GET`` method and filter query parameters.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/products/?criteria\[{nameOfCriterion}\]\[type\]={searchOption}&criteria\[{nameOfCriterion}\]\[value\]={searchingPhrase}'
+
++-----------------+----------------+--------------------------------------------------------------------------+
+| Parameter       | Parameter type | Description                                                              |
++=================+================+==========================================================================+
+| Authorization   | header         | Token received during authentication                                     |
++-----------------+----------------+--------------------------------------------------------------------------+
+| nameOfCriterion | query          | *(required)* The name of criterion (for example "search", "couponBased") |
++-----------------+----------------+--------------------------------------------------------------------------+
+| searchingPhrase | query          | *(required)* The searching phrase                                        |
```

Comment:
I know it's not the best place to suggest that, but why isn't it just `searchPhrase`?